docs/sphinx/source/whatsnew/v0.9.0.rst

```diff
@@ -100,7 +100,8 @@ Bug fixes
 * Pass weather data to solar position calculations in
   :ref:meth:`~pvlib.modelchain.ModelChain.prepare_inputs_from_poa`.
   (:issue:`1065`, :pull:`1140`)
-
+* Reindl model fixed to generate sky_diffuse=0 when GHI=0.
+  (:issue:'1153', :pull:'1154', :ghuser:'jsstein')
```

Comment:
```suggestion
  (:issue:`1153`, :pull:`1154`, :ghuser:`jsstein`)
```

Reply:
I was about to merge when I noticed these single quotes need to be back ticks.

Traditionally we've not put contributor names next to individual lines in the what's new file, but I have no objection to it. I guess I'd still put the contributor name at the end, too.

Reply:
I just pushed a formatting correction and removed the contributor info.  Thanks for the review!